src/Course/List.hs

```diff
@@ -214,8 +212,7 @@ flatMap =
 flattenAgain ::
   List (List a)
   -> List a
-flattenAgain =
-  error "todo: Course.List#flattenAgain"
+flattenAgain = flatMap $ id
```

Comment:
don't need `$` here. `$` is just function application (with the first argument applied to the second argument) with low precedence, so here `flattenAgain = flatMap id` is the same thing.